### PR TITLE
Remove expects from middle of tests

### PR DESCRIPTION
**Pull request by iwootten**
### What is the context of this PR?

Remove interim expects from middle of functional tests which serve no purpose. Currently one such expect is unnecessarily testing an age.

### How to review 

Check the tests continue to pass.

### CODE REVIEW

File: tests/functional/spec/features/repeating_sections/repeating_sections_with_hub_and_spoke.spec.js [hunk 1]
```diff
@@ -214,8 +209,6 @@ describe("Feature: Repeating Sections with Hub and Spoke", () => {
       $(ProxyPage.yes()).click();
       $(ProxyPage.submit()).click();
 
-      expect($(DateOfBirthPage.questionText()).getText()).to.equal("What is Samuel Clemens’ date of birth?");
-
```
Comment (ajmaddaford) [inline tests/functional/spec/features/repeating_sections/repeating_sections_with_hub_and_spoke.spec.js:218]: These tests are testing that the question text is proxy or not, so we should keep this expect and remove the rest of the lines of this test. I don't think we need to check two separate pages for the proxy text.

File: tests/functional/spec/features/repeating_sections/repeating_sections_with_hub_and_spoke.spec.js [hunk 2]
```diff
@@ -194,15 +193,11 @@ describe("Feature: Repeating Sections with Hub and Spoke", () => {
       $(ProxyPage.noIMAnsweringForMyself()).click();
       $(ProxyPage.submit()).click();
 
-      expect($(DateOfBirthPage.questionText()).getText()).to.equal("What is your date of birth?");
-
```
Comment (ajmaddaford) [inline tests/functional/spec/features/repeating_sections/repeating_sections_with_hub_and_spoke.spec.js:198]: These tests are testing that the question text is proxy or not, so we should keep this expect and remove the rest of the lines of this test. I don't think we need to check two separate pages for the proxy text.

File: tests/functional/spec/features/repeating_sections/repeating_sections_with_hub_and_spoke.spec.js (hunk 1 above)
Comment (iwootten) [reply tests/functional/spec/features/repeating_sections/repeating_sections_with_hub_and_spoke.spec.js:218]: As discussed, each section needs to be completed for subsequent tests to follow. We can only therefore remove the interim expects.

**Comment by iwootten** [comment]
[Bug: Functional test failure due to date based test](https://trello.com/c/dCsNqbya/3993-bug-functional-test-failure-due-to-date-based-test)

File: tests/functional/spec/features/repeating_sections/repeating_sections_with_hub_and_spoke.spec.js (hunk 1 above)
Comment (ajmaddaford) [reply tests/functional/spec/features/repeating_sections/repeating_sections_with_hub_and_spoke.spec.js:218]: 🤦

**Pull request merged**